step_05/ReadMe.md

```diff
@@ -1,21 +1,20 @@
 > You can download the code of this step [here](../src/step_05.py) or all the steps [here](https://github.com/Avaiga/taipy-getting-started-core/tree/develop/src).
 
-# Step 5: Scopes
+# Scopes
 
-[Scopes](https://docs.taipy.io/en/latest/manuals/core/concepts/scope/) determine how Data Nodes are shared between cycles, scenarios, and pipelines. The developer may decide to:
+*Time to complete: 15min; Level: Intermediate*
+Requirements: Cycle, Scenario, Data Node, Task
 
-- Keep Data Nodes local to each pipeline.
+[Scopes](https://docs.taipy.io/en/latest/manuals/core/concepts/scope/) determine how Data Nodes are shared between cycles, scenarios, and pipelines. The developer may decide to:
 
-- Extend the scope by sharing data nodes between a given scenario's pipelines.
+- Having one data node for each scenario.
 
 - Extend the scope by sharing data nodes across all scenarios of a given cycle.
 
-- Finally, extend the scope globally (across all scenarios of all cycles). For example, the initial/historical dataset is usually shared by all the scenarios/pipelines/cycles. It has a Global Scope and will be unique in the entire application.
+- Finally, extend the scope globally (across all scenarios of all cycles). For example, the initial/historical dataset is usually shared by all the scenarios/pipelines/cycles. It has a Global Scope and is unique in the entire application.
 
 To summarize, the different possible scopes are:
 
-- _Pipeline scope_: two pipelines can reference different Data Nodes even if their names are the same. For example, we can have a _prediction_ Data Node of an ARIMA model (ARIMA pipeline) and a _prediction_ Data Node of a RandomForest model (RandomForest pipeline). A scenario can contain multiple pipelines.
-
 - _Scenario scope (default)_: pipelines share the same Data Node within a scenario. 
 
 - _Cycle scope_: scenarios from the same Cycle share the same Data Node.
```

Comment:
I suggest rewriting this to 3 bullet points. E.g.

```markdown
- `Scope.SCENARIO` (default): Explanation...

- `Scope.CYCLE`: ...

- `Scope.GLOBAL`: ...
```

You can also remove line 24 since you have already noted the default Scope.